geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/InternalDistributedSystemJUnitTest.java

```diff
@@ -39,8 +39,11 @@
 import static org.apache.geode.distributed.ConfigurationProperties.STATISTIC_SAMPLING_ENABLED;
 import static org.apache.geode.internal.AvailablePortHelper.getRandomAvailableTCPPort;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
I think `org.assertj.core.api.Assertions.assertThat` should be used for these instead of mixing two assertion libraries.

You can also easily replace `org.junit.Assert.fail` with `org.assertj.core.api.Assertions.fail` just by swapping the import. Then that just leaves `org.junit.Assert.assertEquals` if you want to replace it with AssertJ.

Reply:
Good call! I have to fix some other tests so I will do this in my next push!

Reply:
Check it out now! Updated to JUnit 5, uses annotations for System Properties, use assertj, and merge duplicate test files into one. Feels so much cleaner.